templates/zerver/help/change-your-name.md

```diff
@@ -15,7 +15,7 @@ is useful when users' names are managed via LDAP or another data source.
 
 1. Under **User settings**, click on the button with your name in it.
 
-1. Change your name, and click **Save**.
+1. Enter your new name, and click **Change**.
```

Comment:
This made be realize that the user documentation needs to be updated in accordance with the changes made within #17947 and #18232. 
For example, the <a href = 'https://zulip.com/help/delete-a-topic#delete-a-topic'>delete-topic</a> documentation should have the text as `Confirm` instead of `Delete topic`.

Reply:
Resolving this as it is fixed by #18364.